File: 1.Python/Program20.py
# 8. Write a program which accept one number and display below pattern.
# Input :  5
# Output :  1
#           1 2
#           1 2 3
#           1 2 3 4
#           1 2 3 4 5
def Pattern(No):
     i=0
     j=0
     for i in range(1,No+1):
         for j in range(1,No+1):
             if i>=j:
                print(j,end=' ')
         print()

def Main():
   No=0
   print("Enter the one number:")
   No =int(input())
   Pattern(No)

if __name__ == "__main__":
   Main()

File: 0.C_CPP_JAVA Programs/JAVA/Program20.java
/*
5. Write a java program which accept string from user and display it in reverse order.
Input :   “MarvellouS”
Output :   “SuollevraM”
 */
import java.lang.*;
import java.util.*;
class DemoString
{
    void Pattern(String str)
    {
      char arr1[]=str.toCharArray();
      String RevStr="";
      int i=str.length()-1;
      while (i>=0)   
      {
          RevStr=RevStr+ str.charAt(i);
          i--;
      }
      System.out.println("Your Reserse string is:" + RevStr);
    }
}
class Demo
{
    public static void main(String arg[])
    {

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the first String:");
        String str1=sobj.next();

        DemoString dobj=new DemoString();

         dobj.Pattern(str1);
    }
}